docs/documentation/sitespeed.io/developers/index.md

```diff
@@ -101,6 +101,10 @@ Where pageInfo is the data structure that you wanna inspect.
  
  We love pull requests and before you make a big change or add functionality, please open an issue proposing the change to other contributors so you got feedback on the idea before take the time to write precious code!
 
+#### Committing changes
+ * Install Commitizen
```

Comment:
If you can add a full example like npm install ... that's good, because there are people that aren't so used to node that wants to contribute!

Reply:
will add that :+1: